tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/im2latex_evaluator.py

```diff
@@ -0,0 +1,302 @@
+"""
+Copyright (c) 2020 Intel Corporation
```

Comment:
Please align copyright
```suggestion
Copyright (c) 2018-2020 Intel Corporation
```